<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_1</class>
 <widget class="QDialog" name="fct_comp_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nom, Prénom, pays, catégorie des sportifs d'une équipe</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Saisissez un numéro d'équipe :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Travail à réaliser : Ajouter la date de naissance des sportifs de l'équipe</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="table_fct_comp_1">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>110</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>nomSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>prenomSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pays</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>categorieSp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateNaisSp</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_fct_comp_1">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_1</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
